examples/khr-3hv/khr-3hv_continuous/controllers/robot_supervisor_manager/robot_supervisor_manager.py

```diff
@@ -0,0 +1,39 @@
+import PPO_runner
+from dataclasses import dataclass
+
+"""
+Modify these parameters if needed.
+"""
+PROJECT_NAME = "KHR-3HV Humanoid"  # The project name on wandb
+WANDA_SAVE_PERIOD = 100  # Logging period (steps) on wandb
```

Comment:
Should probably be `WANDB_SAVE_PERIOD`.

```suggestion
WANDB_SAVE_PERIOD = 100  # Logging period (steps) on wandb
```

Reply:
Hahaha, Scarlet Witch (Wanda) 🤣

Reply:
Haha :laughing: